src/webinars/2024/bringing-ai-to-nodered.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Bringing AI to Node-RED with FlowFuse Assistant
+subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
+image:
+date: 
+time: 
+duration: 60
+hosts: ["nick-oleary"]
+hubspot:
+    formId:
+---
+
+**In this webinar, FlowFuse CTO and Node-RED creator Nick O'Leary will show how AI is being brought into the Node-RED world within FlowFuse.**
+
+<!--more-->
+
+There's no denying the place AI has in the software development world. Helping users be more productive and creating richer solutions using the power of AI.
+
```

Comment:
Add a short introduction of what FlowFuse Assistant is and what it enables, what you can achieve etc.

Reply:
Have expanded a bit more on the FlowFuse Assistant.